README.md

```diff
@@ -1,57 +1,59 @@
 
-Your database will be PSQL, and you will interact with it using [node-postgres](https://node-postgres.com/).
+This API is using PSQL, and interaction with it using [node-postgres](https://node-postgres.com/).
 
-## Kanban
+### 1. Setting up the Environment
 
-### Link to your Trello Board here: https://trello.com/b/gBtX4GN3/be-nc-games
+Before running the application or tests, you should ensure that all dependencies are installed by running the following command:
 
-To keep track of the tasks involved in this project we're going to use a kanban board. Ensure that you work on one _ticket_ at time. You can click on the ticket to find out more information about what is required for the feature. A ticket is not considered complete unless both the happy path and errors response are handled. You can make use of the checklist on each ticket to keep track of the errors you want to handle. You can also make use of [error-handling.md](error-handling.md) to consider the error codes we may wish to respond with.
-
-**Please ensure you work through the tickets in numerical order.**
+```
+npm install
+```
 
-## Git Branching and Pull Requests
+This will install all required dependencies for the project.
 
-You will be working on each ticket on a new **branch**.
+### 2. Setting up the Environment Variables
 
-To create and switch to a new git branch use the command:
+To run this project locally, you will need to set up environment variables for the two databases. The necessary environment variables are defined in two files named ***.env.development*** and ***.env.test***.
 
-```
-git checkout -b <new branch name>
-```
+Before setting up the environment variables, you should first make a copy of the example environment file ***.env.example*** and rename it to ***.env.development*** and ***.env.test*** respectively.
 
-This will create a branch and move over to that branch. (Omit the `-b` flag if you wish to switch to an already existing branch).
+The **PGDATABASE** environment variable defines the name of the database that the application will use for each environment. The difference between the development and test databases is that the development database is used to store data during the development process, while the test database is used to run automated tests against the application.
 
-We recommend that you name the branch after the number assigned to each ticket via the header. eg. `ncnews-1`
+Next, open the ***.env.development*** and ***.env.test*** files and replace database_name_here with the name of your databases for each environment variable **PGDATABASE**.
 
-When pushing the branch to git hub ensure that you make reference to the branch you are pushing to on the remote.
+Here's what the content of ***.env.development*** and ***.env.test*** files should look like:
 
+***.env.development***
 ```
-git push origin <branch name>
+PGDATABASE=nc_games
```

Comment:
You should not reveal the name of your database for security reasons. Please replace this with a placeholder.